### ISSUE TITLE
Migrate `no-dupe-v-else-if`

### ISSUE THREAD

**Issue by Shinigami92** [Shinigami92/eslint-plugin-vue-pug-sfc#9]
https://eslint.vuejs.org/rules/no-dupe-v-else-if

**Comment by Shinigami92** [comment]
@TokugawaTakeshi
Prioritize next due to: https://github.com/TokugawaTakeshi/vue-eslint-pug_support_test/blob/7a08beda5096f9ac97c8a4077387844a72a584ec/src/App.vue#L6-L8

**Comment by Shinigami92** [comment]
Damn! @TokugawaTakeshi I started implementing this rule today and this is really hardcore to implement 😬

I'm not sure if I'm able to support this fully and exactly as the original rule, cause I don't have access to `node`s in expressions like the HTML parser does.
So yeah there is some stuff possible to do like splitting the string and check the logic but just saying that I'm not sure if it is worth to implement it that deep.

Example:

```pug
div(v-if="a")
div(v-else-if="b && c")
div(v-else-if="d && (c && e && b || a)")
//-                  ~~~~~~~~~~~~~~~~ this should be detected
```

I would need to check the attributes before and `b && c` is same as `c && e && b || a`
And then the plugin needs to set the range from the `c` to `a` within the `v-else-if`-token's _value_

I think it sounds easier than it is...

I think I will implement this rule first for basic stuff like detecting same values and then I will implement the harder stuff (much) later

So the following will be detected because the value is exactly the same (`booleanField1`):

```pug
div(v-if="booleanField1") FOO
div(v-else-if="booleanField1") BAR
```
